conf/messages

```diff
@@ -4499,13 +4499,13 @@ viewReturn.changeReturn=Change return
 # MULTIPLE DISPOSALS EXAMPLE PROPERTY PAGES - START
 #===================================================
 
-multipleDisposalsDisposalPrice.title=What was the disposal value of the property?
-multipleDisposalsDisposalPrice.personalRep.title=What was the disposal value of the property?
-multipleDisposalsDisposalPrice.capacitor.title=What was the disposal value of the property?
-multipleDisposalsDisposalPrice.agent.title=What was the disposal value of the property?
-multipleDisposalsDisposalPrice.trust.title=What was the disposal value of the property?
-multipleDisposalsDisposalPrice.personalRepInPeriodOfAdmin.title=What were the proceeds of the disposal?
-multipleDisposalsDisposalPrice.personalRepInPeriodOfAdmin.agent.title=What were the proceeds of the disposal?
+multipleDisposalsDisposalPrice.title=Property disposal value
```

Comment:
can probably collapse this into a single entry

Reply:
I did respond to that under the previous PR. Here's the content of the comment: "I thought so too but that's the way its done in the message file so I adjusted to that to maintain consistency"

Reply:
Although not strictly needed for this ticket, we do want to try and minimise duplicate/unneeded code. Lukasz and I did collapse these in our PRs